spec/features/edit_page_spec.rb

```diff
@@ -35,4 +35,26 @@
 
     expect(page).to have_content("true")
   end
+
+  it "displays selectable strings as dropdowns", :js do
+    customer = create(:customer, kind: :standard)
+
+    visit edit_admin_customer_path(customer)
+
+    select "vip", from: "Kind"
+
```

Comment:
We haven't been putting any newlines between the action lines of our test cases. Would you mind removing lines 43 and 45? Thanks!